packages/calling-component-bindings/src/handlers/createCommonHandlers.ts

```diff
@@ -51,6 +51,14 @@ export interface CommonCallingHandlers {
   onStopScreenShare: () => Promise<void>;
   onToggleScreenShare: () => Promise<void>;
   onHangUp: (forEveryone?: boolean) => Promise<void>;
+  /* @conditional-compile-remove(raise-hands) */
+  onRaiseHand: () => Promise<void>;
+  /* @conditional-compile-remove(raise-hands) */
+  onLowerHand: () => Promise<void>;
+  /* @conditional-compile-remove(raise-hands) */
+  onLowerHands: (userId: string[]) => Promise<void>;
+  /* @conditional-compile-remove(raise-hands) */
```

Comment:
Is that is onToggle style then we just have that without onRaise/onLower

or we just use two function styles ( onRaise/onLower) without onToggle

(just choose one style of those)